week1-IntroAndSetupGuide.md

```diff
@@ -0,0 +1,38 @@
+# Part 1
+This chapter will get you setup on your repository, and kick start your front end code.
+
+## Setting up
+
+As a preparation for this session you should have received a repository name (if you don't, let an instructor know), have git and some git client installed ([GitHub Desktop](https://desktop.github.com/) recommended). Also, please make sure you have read and completed all the [Requirements](https://github.com/codessintheclassroom/classroom-material/blob/master/REQUIREMENTS.md) for this workshop.
+
+With your repo name in hands (it will be your working repo for the duration of the project), let's get started! 
+
+ 1. Create a local folder to keep your code. Ex: C:\CodessInTheClassroom.
+ 2. Find your repo at [CodessInTheClassrom GitHub project](https://github.com/codessintheclassroom).
+ 3. `git clone` your repository.  
+   3.1. If you're using GitHub desktop, see [this page](https://help.github.com/en/desktop/contributing-to-projects/cloning-a-repository-from-github-desktop) for further instructions on cloning.
+
+At this stage you have your code checked out locally, and by default you would be working off the `master` branch. We should avoid committing and pushing code straight to `master`, as this branch is usually where your stable code should be (we normally use `master` as a source for production code). We'll talk about this in a bit more detail when we discuss Pull Requests. 
+Having that in mind, we will now:
+
+ 1. Create a new branch for the next coding steps.    
+  1.1. Make sure you have a descriptive branch name. Ex: `your-username/workshop1-adding-react-app`.  
```

Comment:
add a brief description of how to create a git branch for those that don't use the GitHub Desktop app - because it's not available on Linux, and some people will use Linux